src/scripts/clipperUI/ratingsHelper.ts

```diff
@@ -0,0 +1,538 @@
+import {Localization} from "../localization/localization";
```

Comment:
See previous comment, if we created a RatingsPanelClass, a lot of these methods could live there instead.